security-analysis/security-analysis-api/pom.xml

```diff
@@ -44,6 +44,14 @@
             <groupId>com.fasterxml.jackson.core</groupId>
             <artifactId>jackson-databind</artifactId>
         </dependency>
+        <!-- https://mvnrepository.com/artifact/org.junit.platform/junit-platform-launcher -->
+        <dependency>
+            <groupId>org.junit.platform</groupId>
+            <artifactId>junit-platform-launcher</artifactId>
+            <version>1.10.2</version>
+            <scope>test</scope>
```

Comment:
Is it possible to avoid a hardcoded version ? Either by using a dependency management entry in a parent POM or a variable ?

Reply:
I forgot to remove that 
it is to launch the tests in intellij